pytorch_lightning/loggers/wandb.py

```diff
@@ -93,7 +92,7 @@ def __init__(
         save_dir: Optional[str] = None,
         offline: Optional[bool] = False,
         id: Optional[str] = None,
-        anonymous: Optional[bool] = False,
+        anonymous: Optional[bool] = None,
```

Comment:
Why change the default?

Reply:
This is actually still the same behavior but is more in line with [the W&B documentation](https://docs.wandb.ai/ref/python/init) so less prone to potential bugs.